crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -197,6 +197,10 @@ impl ShaderCache {
             }
         }
 
+        // TODO: Check if this is supported, though I'm not sure if bevy works without this feature?
+        // We can't compile for native at least without it.
+        capabilities |= Capabilities::CUBE_ARRAY_TEXTURES;
```

Comment:
We need this for native to compile, and webgl2 still works even with it enabled.

Ideally we feature gate it but I'm not sure how since it's a part of `DownLevelFlags` and doesn't seem to be exposed as a wgpu feature or limit.

Reply:
if we want to check, we could pass the `RenderAdapter` down through `PipelineCache::new` and use `adapter.get_downlevel_capabilities().flags`. but i think it's ok not to since we already assume support in point shadow bindings anyway.

Reply:
Hmm. Ok. We can fix it if it’s a problem anywhere.